tests/functional/test_to_service_interstitial.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from bs4 import BeautifulSoup
+from flask import url_for
+import pytest
+
+
+def request(url, method):
+    r = method(url)
+    r.soup = BeautifulSoup(r.get_data(as_text=True), 'html.parser')
+    return r
+
+
+class When_on_to_service_interstitial(object):
+
+    @pytest.fixture(autouse=True)
+    def setup_page(self, client, app):
+        self.config_countdown = app.config['META_REFRESH_DELAY']
```

Comment:
This variable could be better named - `config_countdown` sounds like a verb. Perhaps `delay`?